lib/active_model/serializer.rb

```diff
@@ -26,35 +26,12 @@ class << self
     end
 
     def self.inherited(base)
-      base._attributes = self._attributes.try(:dup) || []
-      base._attributes_keys = self._attributes_keys.try(:dup) || {}
```

Comment:
I don't see any case when it makes sense. I treat `ActiveModel::Serializer` class as an abstract class.